maildump_runner/main.py

```diff
@@ -87,11 +91,19 @@ def main():
         args.htpasswd = os.path.abspath(args.htpasswd)
         print('Htpasswd path is relative, using {0}'.format(args.htpasswd))
 
+    if args.smtp_auth and not os.path.isabs(args.smtp_auth):
+        args.smtp_auth = os.path.abspath(args.smtp_auth)
+        print('SMTP Htpasswd path is relative, using {0}'.format(args.smtp_auth))
+
     # Check if the password file is valid
     if args.htpasswd and not os.path.isfile(args.htpasswd):
         print('Htpasswd file does not exist')
         sys.exit(1)
 
+    if args.smtp_auth and not os.path.isfile(args.smtp_auth):
+        print('SMTP Htpasswd file does not exist')
```

Comment:
similar here ("Htpasswd" with this capitalization looks weird when not at the beginning of the sentence)